lib/far_mar/vendor.rb

```diff
@@ -0,0 +1,100 @@
+#lib/far_mar/vendor.rb
+require 'csv'
+
+class FarMar::Vendor #capitalize all names of class
+  attr_reader :id, :vendorname, :numberofemployees, :marketid
+
+  def initialize(attrs)
+
+      @id  =attrs[0].to_i
```

Comment:
There is an extra level of indention from line 9 to 12. Those lines should be out-dented one level.